src/authenticationinapp/authenticationinapplistener.cpp

```diff
@@ -775,6 +780,8 @@ void AuthenticationInAppListener::processRequestFailure(
         return;
       }
 
+      emit MozillaVPN::instance()->recordGleanEventWithStringType(
+          GleanSample::authenticationInappError, "errno", "125");
```

Comment:
To prevent future confusion, I'd like to see this handled slightly differently because we're actually only capturing a subset of 125 errors here. This is "errno 125 + not email-captcha"

Reply:
I like this! we can have errno + extra desc if available.